index.js

```diff
@@ -180,6 +183,10 @@ for (const event of events) {
             const now = performance.now();
             sessionEventLog[event].push({ event: now, actual: e.timeStamp });
         }
+
+        if (["pointerdown", "pointermove", "pointerup"].includes(event)) {
+            JsToDef.send(event, e.timeStamp);
+        }
```

Comment:
Sends the input event to Defold with the `actual` timestamp that the event occurred so as to match the JS equivalent visualizations.